charts/stable/haste-server/values.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: ghcr.io/rwaltr/haste-server
+  pullPolicy: IfNotPresent
+  tag: 2021-04-19
```

Comment:
Should we wait until the kah image is created?

Reply:
We'll do it live